content/applications/general/email_communication/email_servers.rst

```diff
@@ -129,94 +129,80 @@ How to manage inbound messages
-alias in your mail server.
+Depending on your mail server, there might be several methods to fetch emails. The easiest and most
+recommended method is to manage one email address per Odoo alias in your mail server.
 
-* Create the corresponding email addresses in your mail server 
+* Create the corresponding email addresses in your mail server
   (catchall@, bounce@, sales@, etc.).
 * Set your domain name in the General Settings.
 
   .. image:: email_servers/alias-domain.png
       :align: center
+      :alt: Setting the domain alias configuration on Odoo.
 
-* If you use Odoo on-premise, create an *Incoming Mail Server* in Odoo for each alias. 
-  You can do it from the General Settings as well. Fill out the form according 
-  to your email provider’s settings. 
-  Leave the *Actions to Perform on Incoming Mails* blank. Once all the 
-  information has been filled out, click on *TEST & CONFIRM*.
+* If you use Odoo on-premise, create an :guilabel:`Incoming Mail Server` in Odoo for each alias.
+  You can do it from the General Settings as well. Fill out the form according to your email
+  provider's settings. Leave the :guilabel:`Actions to Perform on Incoming Mails` blank. Once all
+  the information has been filled out, click on :guilabel:`TEST & CONFIRM`.
 
-.. image:: email_servers/incoming-server.png
+  .. image:: email_servers/incoming-server.png
     :align: center
+    :alt: Incoming mail server configuration on Odoo.
 
-* If you use Odoo Online or Odoo.sh, We do recommend to redirect incoming messages 
-  to Odoo's domain name rather than exclusively use your own email server. 
-  That way you will receive incoming messages without delay. Indeed, Odoo Online is fetching
-  incoming messages of external servers once per hour only. 
-  You should set redirections for all the email addresses to Odoo's domain name in your 
-  email server (e.g. *catchall@mydomain.ext* to *catchall@mycompany.odoo.com*).
-
-.. tip:: All the aliases are customizable in Odoo.
-   Object aliases can be edited from their  respective configuration view.
-   To edit catchall and bounce aliases, you first need to activate the
-   :ref:`developer mode <developer-mode>`.
+* If you use Odoo Online or Odoo.sh, We do recommend to redirect incoming messages to Odoo's domain
+  name rather than exclusively use your own email server. That way you will receive incoming
+  messages without delay. Indeed, Odoo Online is fetching incoming messages of external servers
+  once per hour only. You should set redirections for all the email addresses to Odoo's domain name
+  in your email server (e.g. *catchall\@mydomain.ext* to *catchall\@mycompany.odoo.com*).
 
- Then go to :menuselection:`Settings --> Technical --> Parameters --> System Parameters`
- to customize the aliases (*mail.catchall.alias* & * mail.bounce.alias*).
+.. tip:: All the aliases are customizable in Odoo. Object aliases can be edited from their
```

Comment:
I think `All the aliases...` needs to be on its own line, not the same line as the `.. tip::`